### PR TITLE
cloud labels can be added to an existing cluster

### PR DESCRIPTION
**Pull request by szuecs**
This adds a section in case of a user wants to modify an existing cluster instead of creating a new one.

### CODE REVIEW

**Comment by christopherhein** [comment]
Thank you @szuecs for this addition - just a couple comments.

**Comment by szuecs** [comment]
@christopherhein what comments do you have?

**Comment by arun-gupta** [comment]
@christopherhein do you have any comments or can we merge this?

**Comment by christopherhein** [comment]
Sorry, I forgot to finish the feedback. Notes added.

File: ingress-controllers/readme.adoc [hunk 1]
```diff
@@ -252,11 +258,29 @@ your S3 Bucket name for kops configuration and you kops cluster name:
...
 The you create the kops cluster and validate that everything is set up properly.
 
-
         export KOPS_STATE_STORE=s3://${S3_BUCKET}
         kops create cluster --name $KOPS_CLUSTER_NAME --zones $AWS_AVAILABILITY_ZONES --cloud-labels kubernetes.io/cluster/$KOPS_CLUSTER_NAME=owned --yes
         kops validate cluster
 
+=== Modify and existing cluster add cloud labels
+
+Next, you modify your existing Kops cluster and update it.
```
[the remark this answers is not in the document]
Comment (szuecs) [reply ingress-controllers/readme.adoc]: Not being native speaker, but your suggestion sounds weird to me:

To modify your existing kops cluster and update it.

**Comment by szuecs** [comment]
@christopherhein I changed as you suggested, thanks for your feedback

File: ingress-controllers/readme.adoc [hunk 2]
```diff
@@ -250,13 +256,31 @@ your S3 Bucket name for kops configuration and you kops cluster name:
...
+```
+ spec:
+   cloudLabels:
+     kubernetes.io/cluster/example.cluster.k8s.local: owned
+```
+
+Update the cluster with the new configuration:
+
+       kops update cluster $KOPS_CLUSTER_NAME --yes
+
```
Comment (chrislovecnm) [inline ingress-controllers/readme.adoc]: So the update will create the new asg, but after that you need to roll that instance grou with a rolling update command. Someone should test to verify the above statement. How about we get the PR in and test, and then update if we need to.

Comment (szuecs) [reply ingress-controllers/readme.adoc]: @chrislovecnm It's done late in the IAM part https://github.com/aws-samples/aws-workshop-for-kubernetes/pull/306/files#diff-bc4c670ed3d071dc1009d0d55f868b1dL334

I thought first change everything and then execute the rolling update.

Comment (chrislovecnm) [reply ingress-controllers/readme.adoc]: Agreed

**Comment by christopherhein** [comment]
Awesome @chrislovecnm & @szuecs 👏 merging

**Pull request merged**